src/components/TaskGroup/TaskGroup.tsx

```diff
@@ -33,6 +34,10 @@ const TaskGroup = ({ taskList, groupTitle, groupId, isHidden }: TaskItemGroupTyp
   const taskQuantity = groups.find((group) => group.groupId === groupId)!.taskList.length;
   const [taskCount, setTaskCount] = useState(taskQuantity);
 
+  useEffect(() => {
+    groupRef.current?.scrollIntoView({ behavior: 'smooth' });
```

Comment:
Is that added for each task group in the list and scrolls one by one to the view during render ?